WalletWasabi.Tests/UnitTests/Crypto/ProofSystemTests.cs

```diff
@@ -196,7 +196,6 @@ public void CanProveAndVerifyBalance(int presentedAmount, int requestedAmount)
 		[InlineData((ulong)uint.MaxValue + 1, 32, false)]
 		[InlineData((ulong)uint.MaxValue + 1, 33, true)]
 		[InlineData(2099999997690000ul, 50, false)]
-		[InlineData(2099999997690000ul, 51, true)]
```

Comment:
No, i deleted because `logBase 2 2099999997690000 = 50.89931074961488` so 51 bits allow numbers greater than the number of bitcoins that will exist. In summary, I think 50 bits is the biggest number we should handle.